core/templates/domain/skill/skill-rights-backend-api.service.ts

```diff
@@ -44,7 +44,10 @@ export class SkillRightsBackendApiService {
     private http: HttpClient,
     private urlInterpolationService: UrlInterpolationService) {}
 
-  _fetchSkillRights(skillId:string, successCallback, errorCallback): void {
+  _fetchSkillRights(
+      skillId:string,
+      successCallback: (value?: Object | PromiseLike<Object>) => void,
+      errorCallback: (reason?: any) => void): void {
```

Comment:
Can we change `reason?: any` to `reason?: string`, as we are calling this function using only string values.

Reply:
Done!